cmd/ocm/cluster/create/create.go

```diff
@@ -29,11 +31,14 @@ import (
 )
 
 var args struct {
-	parameter []string
-	header    []string
-	region    string
-	version   string
-	flavour   string
+	parameter         []string
+	header            []string
+	region            string
+	version           string
+	flavour           string
+	expirationTime    string
+	expirationSeconds time.Duration
+	expiration        *metav1.Time
```

Comment:
I'm not sure `expiration` should be part of `args`, since it's only ever assigned and used inside `run()` we might as well define it locally.